extensions/twitch.js

```diff
@@ -0,0 +1,297 @@
+// Name: Twitch
+// ID: twitch
+// Description: Blocks that allow you to stream easier on TurboWarp.
+// By: BOP <https://scratch.mit.edu/users/CEFSharp/>
+
+(function (Scratch) {
+  "use strict";
+
+  let chat = null;
+  let token = null;
+  let text = null;
+  let image = null;
+
+  class Twitch {
+    getInfo() {
+      return {
+        id: "twitch",
+        name: "Twitch",
+        color1: "#772CE8",
+        menuIconURI: "https://tekinical.github.io/twitchURI.png",
+        docsURI:
+          "https://github.com/Tekinical/tekinical.github.io/blob/main/Twitch%20Extension%20Doc.md",
+        blocks: [
+          {
+            blockType: Scratch.BlockType.LABEL,
+            text: "Twitch Chat",
+          },
+          {
+            opcode: "setupchat",
+            text: "setup chat [username] chat fade enabled [fade] show bots [bots] prevent clipping [prevent] width [width] height [height] position [position]",
+            blockType: Scratch.BlockType.COMMAND,
+            blockIconURI: "https://tekinical.github.io/twitchURI.png",
+            arguments: {
+              username: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "Twitch Username",
+              },
+              fade: {
+                type: Scratch.ArgumentType.NUMBER,
+                defaultValue: 30,
+              },
+              bots: {
+                type: Scratch.ArgumentType.BOOLEAN,
+                defaultValue: true,
+              },
+              prevent: {
+                type: Scratch.ArgumentType.BOOLEAN,
+                defaultValue: false,
+              },
+              width: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "100%",
+              },
+              height: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "100%",
+              },
+              position: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "absolute",
```

Comment:
Can this be a menu (dropdown)?

Reply:
![image](https://github.com/TurboWarp/extensions/assets/140775902/f36a495d-414b-4765-a7ae-544f14d2d976)